File: EPSILON/SAMPLE FILE/E-publishing/Tools/Altova/Altova/XMLSpy2006/spl/java/altova/text/edifact/EDIX12Settings.java
[call GenerateFileHeader("EDIX12Settings.java")]
package com.altova.text.edifact;

public class EDIX12Settings extends EDISettings {
	private char m_SubElementSeparator = '!';

	private String m_InterchangeControlVersionNumber = "05012";

	private boolean m_RequestAcknowledgement = true;

	public char getSubElementSeparator() {
		return m_SubElementSeparator;
	}

	public void setSubElementSeparator(char rhs) {
		m_SubElementSeparator = rhs;
	}

	public String getInterchangeControlVersionNumber() {
		return m_InterchangeControlVersionNumber;
	}

	public void setInterchangeControlVersionNumber(String rhs) {
		m_InterchangeControlVersionNumber = rhs;
	}

	public boolean getRequestAcknowledgement() {
		return m_RequestAcknowledgement;
	}

	public void setRequestAcknowledgement(boolean rhs) {
		m_RequestAcknowledgement = rhs;
	}
}

File: EPSILON/TOOLS/Altova/Altova/XMLSpy2006/spl/cpp/AltovaText/EDIX12Settings.cpp
[call GenerateFileHeader("EDIX12Settings.cpp")]
#include "StdAfx.h"
#include "EDIX12Settings.h"

namespace altova
{
namespace text
{
namespace edi
{

CEDIX12Settings::CEDIX12Settings()
:	m_SubElementSeparator(_T('!'))
,	m_InterchangeControlVersionNumber(_T("05012"))
,	m_RequestAcknowledgement(true)
{}
TCHAR CEDIX12Settings::GetSubElementSeparator() const
{
    return m_SubElementSeparator;
}

void CEDIX12Settings::SetSubElementSeparator(TCHAR rhs)
{
    m_SubElementSeparator = rhs;
}

const tstring& CEDIX12Settings::GetInterchangeControlVersionNumber() const
{
    return m_InterchangeControlVersionNumber;
}

void CEDIX12Settings::SetInterchangeControlVersionNumber(const tstring& rhs)
{
    m_InterchangeControlVersionNumber = rhs;
}

bool CEDIX12Settings::GetRequestAcknowledgement() const
{
    return m_RequestAcknowledgement;
}

void CEDIX12Settings::SetRequestAcknowledgement(bool rhs)
{
    m_RequestAcknowledgement = rhs;
}


} // namespace edi
} // namespace text
} // namespace altova
